### ISSUE TITLE
Consult some details

### ISSUE THREAD

**Issue by baifanxxx** [ChexuanQiao/Pixelwise_Knowledge_Distillation_PSPNet18#1]
Hi author, 

Thanks  for your intetesting work. It is very useful for me. However, I want to consult some details for it. For example, in you readme, 'T1.8 A0.2' means temperature=1.8 and alpha=0.2 in config? And mIoU/mAcc/aAcc=0.7130/0.7904/0.9505 is based on students PSPNet18 teacher PSPNet50 ? Another more, when I train student PSPNet18 with teacher PSPNet50, it has more GPU occupy than only PSPNet18, it may be close to even more GPU occupy with only PSPNet50. It is reasonable ? 

Thank you,
Best regards.

**Comment by ChexuanQiao** [comment]
Hi baifanxxx,

With regard to your questions,

1. Yes, 'T1.8 A0.2' means temperature=1.8 and alpha=0.2;
2. Yes, mIoU/mAcc/aAcc=0.7130/0.7904/0.9505 is based on student PSPNet18 network and teacher PSPNet50 network;
3. Knowledge distillation introduces a second network during training, so do expect a considerable increase in GPU occupancy.

Regards.

**Comment by baifanxxx** [comment]
Hi author,

Thank you for your reply. Could you upload your config file about cityscapes_kdnet18.yaml? I always had a bad results. I don't know whether my hyperparameter is wrong or not.  Given that:
`main_loss = 0.2*ce_loss + 0.8*kl_loss `
`loss = main_loss + args.aux_weight * aux_loss`
where args.aux_weight = 0.4

Thank you again.
Best regards.
BAI Fan
